parachain/src/lib.rs

```diff
@@ -95,6 +94,12 @@ pub struct ValidationResult {
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize, Debug))]
 pub struct Id(u32);
 
+/// Type for determining the active set of parachains.
+pub trait ActiveThreads {
+	/// Return the current ordered set of `Id`s of active parathreads.
+	fn active_threads() -> Vec<Id>;
```

Comment:
```suggestion
	fn active_threads() -> Parathreads;
```

suggest adding type alias `pub type Parathreads = Vec<Id>;`